blocks/default-output-block/output-types/default.jsx

```diff
@@ -100,20 +100,20 @@ const SampleOutputType = ({
 		resizerURL,
 		facebookAdmins,
 		nativoIntegration,
-		chartBeatAccountId,
-		chartBeatDomain,
+		chartbeatAccountId,
+		chartbeatDomain,
```

Comment:
if we change this, we may need to make a note of it to all of the clients that might be using the old casing...

Reply:
This casing matches what is in our manifests (https://github.com/WPMedia/arc-themes-manifests/blob/4a57dd074912862449b1e9c540e52d8b79dabfcb/src/manifests/news/1.24.json#L1061 and https://github.com/WPMedia/arc-themes-manifests/blob/4a57dd074912862449b1e9c540e52d8b79dabfcb/src/manifests/news/2.0.3.json#L1049), so right now this shouldn't be working for anybody. From what I can find it seems clients actually using chartbeat (like Irish Times) have ejected the default output type block and made their own, using this casing for the properties in their custom code.